docs/src/api.md

```diff
@@ -167,8 +221,74 @@ PulseDesigner.spiral_base
 PulseDesigner.EPI_example
 ```
 
+## Spin Representation
+
+### `Mag`
+```@docs
+Mag
+```
+
+### `Spinor`
+```@docs
+Spinor
+```
+
+### `Rx`
+```@docs
+Rx
+```
+
+### `Ry`
+```@docs
+Ry
+```
+
+### `Rz`
+```@docs
+Rz
+```
+
+### `Q`
+```@docs
+Q
+```
+
+### `Un`
+```@docs
+Un
+```
+
+
+## Low Level Processing
+
+### `f32`
+```@docs
+f32
+```
+
+### `f64`
+```@docs
+f64
+```
+
+### `cpu`
+```@docs
+cpu
+```
+
+### `gpu`
+```@docs
+gpu
+```
+
+
 ## Simulation
 
+### `default_sim_params`
+```@docs
+default_sim_params
+```
+
```

Comment:
`default_sampling_params`?

Reply:
The `default_sampling_params` function is not defined in this branch. We shouldn't forget that function in the documentation when we add the KomaMRI subpackages.